vsg/rules/concurrent/rule_007.py

```diff
@@ -32,7 +32,7 @@ def _analyze(self, oFile, oLine, iLineNumber):
 
     def _fix_violations(self, oFile):
         for dViolation in self.violations[::-1]:
-            iLineNumber = dViolation['lineNumber']
+            iLineNumber = utils.get_violation_linenumber(dViolation)
```

Comment:
Shouldn't it be `line_number`, not `linenumber`?

Reply:
I always struggle with how to name things.  I think that makes sense.  I will make the update.